README.md

```diff
@@ -144,8 +144,7 @@ To play this game you need to:
 ### Built With
 This project was built using these technologies.
 * Ruby
-* Rubocop
-* VsCode
+* Rspec
```

Comment:
You also need to include a Testing section where you explain how to run the test suite.